system-tests/projects/no-specs-custom-pattern-vue/babel.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
The diff looks really big because I added a new project to system-tests. I needed a project that had a custom spec pattern but was a Vue project.

You should be safe to ignore any changed files under `system-tests/projects/`